public/static/docs/command-reference/cache/index.md

```diff
@@ -15,8 +15,9 @@ positional arguments:
 
 ## Description
 
-After DVC initialization, a hidden directory `.dvc/` is created to contain the
-[DVC files and directories](/doc/user-guide/dvc-files-and-directories),
+At DVC initialization, a new `.dvc/` directory will be created with the
+`config`, `.gitignore`, among
```

Comment:
may be not for this PR - what do you think about the choice of files to mention? I feel that `.dvc/cache` is even more important to mention? or may be we should not be mentioning them at all - just put a link? 

This choice `config`, `.gitignore` looks very artificial to me, tbh. Especially when we don't explain it.

Reply:
`.dvc/cache` was mentioned originally I think, but it's not created at `dvc init` now. Only until some data is actually put in cache with add, fetch, etc.

> may be we should not be mentioning them at all - just put a link?

We could do this. I don't have a strong opinion. `config` and `.gitignore` seem like the more important ones because a) they're always created (the other ones are optional or get created later than init) and b) they're the ones a user is more likely to look at or even edit manually (along with cache/ dir of course).

Reply:
yeah, probably better remove them ... or rephrase - to mention that this directory is about configs (I at least understand the importance of this) and eventually cache

Reply:
OK, updated!

Reply:
Gonna merge this since you previously approved 🙂